packages/web/graphql/withApollo.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
```

Comment:
I guess this file is based on some default Next configuration for apollo?

Reply:
Mostly. FIgured it was best and extend / customize it as we go.